.eslintrc

```diff
@@ -40,6 +41,11 @@
                 }
             }
         ],
-        "vue/html-self-closing": "off"
+        "vue/html-self-closing": "off",
+        "jest/no-disabled-tests": "warn",
+        "jest/no-focused-tests": "error",
+        "jest/no-identical-title": "error",
+        "jest/prefer-to-have-length": "warn",
+        "jest/valid-expect": "error"
```

Comment:
Do all of these need to be overwritten for Jest?

Reply:
Nope but now it validates that we have nice sane tests, thought it would be nice :)

Reply:
Please use the jest extend instead of writing them all down manually here is what I ment :)

Reply:
Yep, fixed it nicely now!